packages/@react-aria/overlays/src/useOverlayPosition.ts

```diff
@@ -119,9 +119,12 @@ export function useOverlayPosition(props: AriaPositionProps): PositionAria {
         crossOffset
       })
     );
+
+  // eslint-disable-next-line react-hooks/exhaustive-deps
   }, deps);
 
   // Update position when anything changes
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Out of curiosity, is it just not checking in deps? or is deps actually missing some?
Lets add a comment when we // disable so that we know why we're ignoring it. It looks like you've started doing that a bit, so that's good. We should make sure every disable of this rule has an explanation

Reply:
For this one, the linter states " React Hook useCallback was passed a dependency list that is not an array literal" and I tried https://github.com/facebook/react/issues/14476#issuecomment-471199055 w/ the JSON.stringify but it broke a lot of stuff so I opted for disabling it. I'll go ahead and put an explanation though, good idea

Reply:
actually, looking at it again maybe I can do away with the dep array since the useEffect below can have a dep on updatePosition